tests/AnalyticsGdpr/AnalyticsGdprCliTest.php

```diff
@@ -41,15 +41,15 @@
 	// Check the output dir if the generated method is correctly generated.
 	$generatedMeta = file_get_contents(dirname(__FILE__, 3) . '/cliOutput/src/AnalyticsGdpr/AnalyticsGdpr.php');
 
-	$this->assertStringContainsString('class AnalyticsGdpr implements ServiceInterface', $generatedMeta);
-	$this->assertStringContainsString('acf_add_options_page', $generatedMeta);
-	$this->assertStringContainsString('acf_add_local_field_group', $generatedMeta);
-	$this->assertStringContainsString('createAnalyticsPage', $generatedMeta);
-	$this->assertStringContainsString('registerAnalytics', $generatedMeta);
-	$this->assertStringContainsString('createGdprModalPage', $generatedMeta);
-	$this->assertStringContainsString('registerGdprModalSettings', $generatedMeta);
-	$this->assertStringContainsString('prepareGdprModalData', $generatedMeta);
-	$this->assertStringNotContainsString('someRandomMethod', $generatedMeta);
+	expect($generatedMeta)->toBeString()->toContain('class AnalyticsGdpr implements ServiceInterface');
+	expect($generatedMeta)->toBeString()->toContain('acf_add_options_page');
+	expect($generatedMeta)->toBeString()->toContain('acf_add_local_field_group');
+	expect($generatedMeta)->toBeString()->toContain('createAnalyticsPage');
+	expect($generatedMeta)->toBeString()->toContain('registerAnalytics');
+	expect($generatedMeta)->toBeString()->toContain('createGdprModalPage');
+	expect($generatedMeta)->toBeString()->toContain('registerGdprModalSettings');
+	expect($generatedMeta)->toBeString()->toContain('prepareGdprModalData');
+	expect($generatedMeta)->toBeString()->not->toContain('someRandomMethod');
```

Comment:
```suggestion
    expect($generatedMeta)
        ->toBeString()
        ->toContain('class AnalyticsGdpr implements ServiceInterface')
        ->toContain('acf_add_options_page')
        ->toContain('acf_add_local_field_group')
        ->toContain('createAnalyticsPage')
        ->toContain('registerAnalytics')
        ->toContain('createGdprModalPage')
        ->toContain('registerGdprModalSettings')
        ->toContain('prepareGdprModalData')
        ->not->toContain('someRandomMethod');
```

Expectations are great because you can chain them 🙂